src/compile-cache.js

```diff
@@ -0,0 +1,131 @@
+'use strict'
+
+var path = require('path')
+var CSON = require('season')
+var fs = require('fs-plus')
+var _ = require('underscore-plus')
```

Comment:
It looks like this file is required before the module cache is set up in `index.js`.

That means any libraries required here and downstream won't pass through it. It would be nice to address that so there aren't any performance regressions.